### ISSUE TITLE
1.19.2 Forge Bug Bosses cause crashes and disconnects on server side.

### ISSUE THREAD

**Issue by franciscogolen** [BobMowzie/MowziesMobs#745]
Hello i am the dev of Cisco's Rpg who messaged you about crashes on server side. Players report getting and io netty decoder error and while i was unable to get the umvuthi  logs i was able to get one for the ferrous wroughtnaught. Singleplayer seems to be unaffected by this issue.
<img width="462" alt="dcmowzies" src="https://github.com/BobMowzie/MowziesMobs/assets/153466209/55eec405-61cd-4cb3-9114-7bdf8c238180">
[latest(2).log](https://github.com/BobMowzie/MowziesMobs/files/15012655/latest.2.log)

**Comment by BobMowzie** [comment]
I'm sorry, can you please remind me the context here?

That error says something on the server is configured to the resource location "minecraft:?" which is invalid. It makes no mention of mowzies mobs.

**Comment by franciscogolen** [comment]
Unfortunately no this is what the players have provided for this particular issue the player says that they got disconnected with that error message about the resource location when entering a chunk with the ferrous wroughtnaught's structure.

**Comment by franciscogolen** [comment]
I have been unable to replicate the issues myself so i'm not sure if it requires two or more players to occur

**Comment by franciscogolen** [comment]
![image](https://github.com/BobMowzie/MowziesMobs/assets/153466209/1fb9607f-d4f9-40ea-b10c-d88d4ce09cb4)

On two seperate occasions.

**Comment by BobMowzie** [comment]
Ah, so these are their client logs? Is there any message on the server itself?
If the server crashed, there should be server crash log with the information I need.

**Comment by franciscogolen** [comment]
I don't think the server crashed for the ferrous wroughtnaught incident unfortunately but hang on let me see if there is anything i can get from them

**Comment by BobMowzie** [comment]
If you could search the modpack's data, resourcepack, and config files for the string "minecraft:?" that would also be helpful.

**Comment by franciscogolen** [comment]
Yeah for the players said the never crashed and the only logs they have are the ones i provided . There are many many of those any ideas on how to narrow it down?

**Comment by BobMowzie** [comment]
Can you please have the players experiencing the issue message me themselves?

**Comment by franciscogolen** [comment]
Alright np.
